master/internal/db/config.go

```diff
@@ -4,15 +4,18 @@ package db
 func DefaultConfig() *Config {
 	return &Config{
 		Migrations: "file://static/migrations",
+		SSLMode:    sslModeDisable,
 	}
 }
 
 // Config hosts configuration fields of the database.
 type Config struct {
```

Comment:
I'm sortof partial to removing all this and just having `connection_string` as opposed to eventually supporting ever pg option